composer.json

```diff
@@ -35,9 +35,9 @@
     ],
     "require": {
         "php": ">=5.3.2",
-        "illuminate/container": "~4.0",
-        "illuminate/routing": "~4.0",
-        "illuminate/support": "~4.0",
+        "illuminate/container": "~5.1",
+        "illuminate/routing": "~5.1",
+        "illuminate/support": "~5.1",
```

Comment:
As far as I can see It is not working with Laravel4 any more. I dont know, How many people uses Laravel5 in compare with Laravel4.